vignettes/covidHubUtils-overview.Rmd

```diff
@@ -83,27 +87,80 @@ forecasts <- load_latest_forecasts(models = "COVIDhub-ensemble",
                       locations = "US",
                       types = c("point","quantile"),
                       targets = inc_case_targets,
-                      source = "zoltar")
+                      source = "zoltar",
+                      verbose = FALSE,
+                      as_of=NULL,
+                      hub = c("US"))
+
+hosp_data <- paste(0:130, "day ahead inc hosp")
+forecasts_hosp <- load_latest_forecasts(models = "COVIDhub-ensemble",
+                      last_forecast_date = "2020-12-07",
+                      forecast_date_window_size = 6, 
+                      locations = "US",
+                      types = c("point","quantile"),
+                      targets = hosp_data,
+                      source = "zoltar",
+                      verbose = FALSE,
+                      as_of=NULL,
+                      hub = c("US"))
+
+inc_case_targets_death <- paste(1:4, "wk ahead inc death")
+forecasts_death <- load_latest_forecasts(models = "COVIDhub-ensemble",
+                      last_forecast_date = "2020-12-07",
+                      forecast_date_window_size = 6, 
+                      locations = "US",
+                      types = c("point","quantile"),
+                      targets = inc_case_targets_death,
+                      source = "zoltar",
+                      verbose = FALSE,
+                      as_of=NULL,
+                      hub = c("US"))
+
 ```
 
 And here is the data that is returned from this query, note that in addition to the essential forecast data itself, some additional columns with information about the locations are returned.
 
-```{r, echo=FALSE}
+```{r, echo=FALSE, message=FALSE}
 datatable(forecasts,
   extensions = 'FixedColumns',
   options = list(
   dom = 't',
   scrollX = TRUE,
   fixedColumns = list(leftColumns = 2)
 ))
+datatable(forecasts_hosp,
```

Comment:
Maybe we do not need to show forecasts_hosp and forecasts_death in table format.

Reply:
This has been updated